.github/workflows/my-ci-pipeline.yml

```
@@ -0,0 +1,16 @@
+name: Continuous Integration
+on:
+  push:
+    paths:
+      - 'README.md'
+  pull_request:
+    paths:
+      - 'README.md'
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

I believe our docs are out of date on this, but note that [this checkout action is actually on v4](https://github.com/actions/checkout)!

```suggestion
    - uses: actions/checkout@v4
```
Switching over should remove the Node/npm warnings (e.g. "Node.js 16 actions are deprecated.")